packages/sources/gsr/src/endpoint/price.ts

```diff
@@ -9,15 +9,24 @@ import { transport } from '../transport/price'
 
 const inputParameters = new InputParameters(priceEndpointInputParametersDefinition)
 
+// Additional fields are due to lwba endpoint
+type EPResponse = SingleNumberResultResponse & {
```

Comment:
Minor: What does `EP` in `EPReponse` stand for? Could we make this more obvious?

Reply:
"endpoint" iirc, was a carryover from one of the other EAs, will change